### PR TITLE
Feature-CMX-2746: update docs: add more destination_countries list

### PR DESCRIPTION
**Pull request by loilet**
CMX-2746: update docs: add more destination_countries list

### CODE REVIEW

**Review by arcdyc-sinch — APPROVED** [review]

File: spec/openapi.yaml [hunk 1]
```diff
@@ -1481,7 +1481,7 @@ paths:
         
         ## The most basic request has the following structure:
           - ```sender_address``` is case insensitive. When one already exists for this account, you get a error message ```conflict```;
-          - ```destination_countries``` currently we only support ```AU``` and ```GB``` as the destination country;
+          - ```destination_countries``` currently we only support ```AU```, ```AT```, ```HR```, ```CY```, ```EE```, ```FI```, ```DE```, ```IE```, ```IT```, ```LV```, ```LT```, ```LU```, ```MT```, ```NL```, ```PT```, ```SI```, ```ES```, ```CH```, ```SE```, ```DK```, ```NO``` and ```GB``` as the destination country;
```
Comment (paultirta) [inline spec/openapi.yaml]: I wonder if it would be better to keep this list as alphabetical?

Comment (loilet) [reply spec/openapi.yaml]: I had updated. Please help review and Merge it!

Thank you! @paultirta

Comment (paultirta) [reply spec/openapi.yaml]: Still one incorrect. **CH** shouldn't be the last one in the list.

Comment (loilet) [reply spec/openapi.yaml]: Done!

**Review by david-porter-messagemedia — APPROVED** [review]

**Review by paultirta — APPROVED** [review]

**Pull request merged**